### PR TITLE
Full Website Navigation

### PR DESCRIPTION
**Pull request by roodjinocherilus**
In this PR, I have developed a full website with several pages: Home, Calculator, and Quote.

### CODE REVIEW

**Comment by roodjinocherilus** [comment]
In this PR, I have developed a full website with several pages: Home, Calculator, and Quote.

**Comment by topeogunleye** [comment]
> In this PR, I have developed a full website with several pages: Home, Calculator, and Quote.

@roodjinocherilus, Please kindly click on the three dots on your PR description to paste this description there as that is the proper way of adding a PR description.

image

**Review by topeogunleye — COMMENTED** [review]
Hi @roodjinocherilus, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

   - You have submitted a project, that doesn't have a pull request description, kindly add one. See a demo on how to add a PR description below:

pull_request_desc
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

File: src/components/Home.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './home.css';
+
+function Home() {
+  return (
+    <div>
```
Comment (topeogunleye) [inline src/components/Home.js:6]: - [ ] As required please make sure that you follow the general layout should match the given wireframe.
- [ ] Kindly add the `Math Magicians` logo to the left of the header.
- [ ] Kindly make sure that your Navigation bar is aligned to the right as seen in the given wireframe.
- [ ] The text `Welcome To The Homepage` should be aligned to the left and not centered as seen in the given wireframe
- [ ] Your paragraph on the page should be aligned to the left as seen in the given wireframe.

|   |  |
| ------------- | ------------- |
| Your Design | image  |
| Given Wireframe  | image  |

File: src/App.js [hunk 2]
```diff
@@ -1,12 +1,38 @@
...
+            </li>
+            <li>
+              <Link to="/quote">Quote</Link>
+            </li>
+          </ul>
+        </nav>
+
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/calculator" element={<Calculator />} />
```
Comment (topeogunleye) [inline src/App.js:16]: - [ ] There should be a text aligned to the left of the calculator as seen the the given wireframe. The calculator shouldn''t be centered. 
- [ ] The calculator should be aligned to the right and the text besides it aligned to the right of the page.

|   |   |
| ------------- | ------------- |
| Your Design | image  |
| Given Wireframe  | image |

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @roodjinocherilus, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * PR has a good summary :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] thank you for submitting your code for review. I appreciate your effort and creativity in completing the Math magicians: full website milestone. However, I noticed that your branch name is `full`, which is not descriptive enough and does not follow the convention of being named after an action or feature. This convention helps to keep the codebase organized and easy to understand by other developers. It also makes it easier to track the changes and merge the branches without conflicts. A good branch name should be descriptive, concise, and use hyphens to separate words. For example, if you worked on adding a calculator component, you could name your branch `added-calculator` or `feature/calculator`. If you fixed a bug in the navigation bar, you could name your branch `fixed-nav-bar` or `bugfix/nav-bar`. if you finished the full Math Magicians website, you could name your branch `finished-math-magicians-website`

Please try to rename your branch and resubmit your code for review. If you have any questions or need further assistance, feel free to reach out to me. Happy coding! 😊
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — COMMENTED** [review]
## Hi @roodjinocherilus 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that was not approved in your previous weeks therefore kindly create a new pull request so that it may be count as a first review in you dashboard

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@BenMukebo` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request closed without merging**